src/js/components/tasks/createForm.tsx

```diff
@@ -117,7 +117,8 @@ const CreateTaskModal = ({
         if (
           type === 'CREATE_OBJECT_SUCCEEDED' &&
           objectType === OBJECT_TYPES.TASK &&
-          object?.slug
+          object?.slug &&
+          epic?.slug
```

Comment:
Even when not `isContributingFromOrg`, we now want to redirect users to the newly-created task detail page on success. I think it can follow this same pattern, something like:

```js
        if (
          type === 'CREATE_OBJECT_SUCCEEDED' &&
          objectType === OBJECT_TYPES.TASK &&
          object?.slug &&
          epic?.slug
        ) {
          const url = routes.task_detail(project.slug, epic.slug, object.slug);
          history.push(url);
        }
```

Then when we don't have an `epic`, we'll redirect to the epic-less task page -- but I need to define the URLs for that first...